src/Compilers/Core/Portable/SourceGeneration/GeneratorSyntaxWalker.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+
+#nullable enable
+namespace Microsoft.CodeAnalysis
+{
+    internal sealed class GeneratorSyntaxWalker : SyntaxWalker
+    {
+        private readonly ImmutableArray<ISyntaxReceiver> _syntaxReceivers;
+
+        public GeneratorSyntaxWalker(ImmutableArray<ISyntaxReceiver> syntaxReceivers)
+        {
+            _syntaxReceivers = syntaxReceivers;
+        }
+
+        public override void Visit(SyntaxNode node)
+        {
+            foreach (var receivier in _syntaxReceivers)
```

Comment:
>receivier [](start = 25, length = 9)

`receiver`

Reply:
*facepalm.gif*

---
In reply to: [395260815](https://github.com/dotnet/roslyn/pull/42507#discussion_r395260815) [](ancestors = 395260815)